src/librustdoc/html/static/js/main.js

```diff
@@ -67,6 +67,11 @@ function resourcePath(basename, extension) {
             ty: sidebarVars.attributes["data-ty"].value,
             relpath: sidebarVars.attributes["data-relpath"].value,
         };
+        var mobileLocationTitle = document.querySelector(".mobile-topbar h2.location");
```

Comment:
Why do you need JS for that? Wouldn't it be better to generate it as is directly?

Reply:
I tried generating directly and it would have required a significant refactoring in what gets passed to page generation for the sidebar. I figure we can always refine that later if we want.

Reply:
Can you add a fixme then please?